src/Map.js

```diff
@@ -904,6 +902,12 @@ export default class Map {
       this.events.emit(Map.REGISTERED_EVENTS.MOVE_END)
     })
 
+    this.map.on('resize', (e) => {
+      log.warn('resize', e)
+      this.width = e.newSize.x
+      this.height = e.newSize.y
+    })
```

Comment:
@SuokN so `resize` will be triggered at the first shot, right? I mean when we mount and open the map.

Reply:
No, resize will be triggered, if you resize the browser.

Reply:
If this is the case, then I don't think we need to care about it, this is a rare case, at least irrelevant to our bug here now, right? I suppose the initial view is calculated at the first time of the rendering?

Reply:
I think we need to care about it,  because if the browser is resized, the height and width of the map will change, and then when we open organization, planter, wallet getInitialBounds will give wrong result.